xwiki-platform-core/xwiki-platform-store/xwiki-platform-store-filesystem-oldcore/src/test/java/org/xwiki/store/filesystem/internal/DefaultTemporaryAttachmentSessionsManagerTest.java

```diff
@@ -120,41 +123,18 @@ void uploadAttachment() throws Exception
             .thenReturn("42");
         when(part.getSize()).thenReturn(41L);
 
+        doAnswer(invocationOnMock -> {
+            TemporaryAttachmentSession temporaryAttachmentSession = invocationOnMock.getArgument(1);
+            assertEquals(sessionId, temporaryAttachmentSession.getSessionId());
+            return null;
+        }).when(httpSession).setAttribute(eq(ATTRIBUTE_KEY), any(TemporaryAttachmentSession.class));
+
```

Comment:
Can this be simplified using argument captor instead https://www.baeldung.com/mockito-argumentcaptor ?